Controller/AdminSecurityController.php

```diff
@@ -12,29 +12,29 @@
 namespace Sonata\UserBundle\Controller;
 
 use FOS\UserBundle\Model\UserInterface;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\RedirectResponse;
```

Comment:
Yes, but only used in the PHP Doc :)

https://github.com/core23/SonataUserBundle/blob/patch-4/Controller/AdminSecurityController.php